hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -392,6 +406,11 @@ private void handleException(BlockOutputStreamEntry streamEntry,
       IOException exception) throws IOException {
     Throwable t = HddsClientUtils.checkForException(exception);
     Preconditions.checkNotNull(t);
+    boolean containerExclusionException = checkIfContainerToExclude(t);
+    if(containerExclusionException) {
```

Comment:
Format - missing space before the brackets `if(containerExclusionException)` should be `if (containerExclusionException)`

Reply:
done